.github/workflows/ci.yaml

```diff
@@ -111,3 +111,24 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
+
+  lint:
+    runs-on: ubuntu-20.04
+    name: Run eslint
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "13.x"
+      - run: npm i
+      - run: npm run lint
+  tsc:
```

Comment:
I think we can have one job for both linting and building, in Rust it just takes longer to do it sequentially, so I split it into multiple jobs.